tests/core/test_float_builtins.out

```diff
@@ -2,7 +2,7 @@
 0.567800 : 1.010100 : 21.320100
 0.123400 : 0.101010 : 12.012300
 0.123400 : 0.101010 : 12.012300
-0.304824 : 1.001016 : 104168858480774546980864.000000
+0.304824 : 1.001016 : 104168858480774530203648.000000
```

Comment:
Why did this change?

Reply:
Good question... do you have any idea?   Do you know how to tell which one is correct?   I didn't look into it much yet.

Reply:
I'm not sure. Running the testcase in gcc I get a slightly different result from both of these, which suggests to me that there isn't an absolute precision that libc is required to provide. Or that if there is, this doesn't regress us any further at least. Also, there are no emscripten changes to `pow.c`, so this is likely either an innocuous musl change, or an improvement. So seems good to me.

Reply:
fwiw, on Alpine Linux (which uses musl too) this is also different.

<details>
  <summary>Details</summary>

```bash
$ docker run -v $(pwd):/app -w /app -it alpine:latest \
   apk add build-base && \
   gcc tests/core/test_float_builtins.c -lm && \
   ./a.out > tests/core/test_float_builtins.out
$ git diff HEAD tests/core/test_float_builtins.out
```

```diff
diff --git a/tests/core/test_float_builtins.out b/tests/core/test_float_builtins.out
index 987a82b9d..2cfbde14e 100644
--- a/tests/core/test_float_builtins.out
+++ b/tests/core/test_float_builtins.out
@@ -2,8 +2,8 @@
 0.567800 : 1.010100 : 21.320100
 0.123400 : 0.101010 : 12.012300
 0.123400 : 0.101010 : 12.012300
-0.304824 : 1.001016 : 104168858480774530203648.000000
-0.000000 : 2.421390 : 101648962858951235671184182497867417669061891127967232769386628692937206561802464554722378383360.000000
+0.304824 : 1.001016 : 104168858480774568304640.000000
+0.000000 : 2.421390 : 101648962858951233521793526030185540119025481904197943243062538336984236579371061407834909442048.000000
 inf : 3.091515 : 3.091515 : 1
 sqrt(x) = 4.000000
 ceil(x) = 16.000000
```
</details>

Reply:
I get the same results as your alpine linux test on my desktop too, using both gcc and clang.  However if I switch to -O2 then gcc changes but clang doesn't (for the final line).   

Given that our results are *already* different to the desktop gcc/clang results I think we can let this slide.

Reply:
I updated my comment because the test ran on my own system. It seems that Alpine also differs in the `asin(x)` test.

> Given that our results are _already_ different to the desktop gcc/clang results I think we can let this slide.

Indeed, I don't think it's worth investigating this further.